### ISSUE TITLE
wptrunner failing to run tests in non-Firefox browsers

### ISSUE THREAD

**Issue by GPHemsley** [w3c/web-platform-tests#6447]
Chrome, Safari, and Edge all fail to run tests, giving this message instead:

```
Traceback (most recent call last):

  File "/home/travis/build/w3c/web-platform-tests/tools/wptrunner/wptrunner/testrunner.py", line 125, in start_runner

    executor = executor_cls(browser, **executor_kwargs)

TypeError: __init__() got an unexpected keyword argument 'group_metadata'
```
This appears to be from the change made by @jgraham 3 days ago (which was committed with [ci skip] and no PR):
https://github.com/w3c/web-platform-tests/commit/8eadce2c378c5380fb3199338a440cfc46b0a03e

In addition, for some reason, these Python exceptions are not failing the builds, which could be giving casual observers false confidence in their changes.

**Comment by gsnedders** [comment]
Fixed in #6450. Is there any reason why we didn't catch this earlier?

**Comment by GPHemsley** [comment]
Well, failing the build upon exception would have made it abundantly clear much sooner.
